packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.test.js

```diff
@@ -696,9 +696,8 @@ describe('The Node SSR Environment', () => {
             assertions: (res) => {
                 expect(res.statusCode).toBe(404)
 
-                // we'll expect that this method is called three times,
-                // twice for rendering pipeline because of the prepass step and one for http request logging
-                expect(console.log).toHaveBeenCalledTimes(3)
+                // Because of the prepass step we'll expect that this method is called twice.
+                expect(console.log).toHaveBeenCalledTimes(2)
```

Comment:
I am wondering why it reduced from 3 to 2 here? 🤔

Reply:
I had the same question about why we increased to 3 on this commit https://github.com/SalesforceCommerceCloud/pwa-kit/commit/f0c26f634c52156642f808d48b300c81f49affec

Do you remember why we increased?  It used to be 2 since we implemented prepass.